updating qbic-pipelines version (#2)
qbic-pipelines/sradownloader: docs/usage.md

The typical command for running the pipeline is as follows:

```bash

> nextflow run -r master steffenlem/sradownloader --run_acc_list '<path_to_acc_list.txt>' --ngc '<path_to_key.ngc>' -profile docker

Author: ```suggestion
nextflow run -r master qbic-pipelines/sradownloader --run_acc_list '<path_to_acc_list.txt>' --ngc '<path_to_key.ngc>' -profile docker
```

When you run the above command, Nextflow automatically pulls the pipeline code from GitHub and stores it as a cached version. When running the pipeline after this, it will always use the cached version if available - even if the pipeline has been updated since. To make sure that you're running the latest version of the pipeline, make sure that you regularly update the cached version of the pipeline:

```bash

> nextflow pull steffenlem/sradownloader

Author: ```suggestion
nextflow pull qbic-pipelines/sradownloader
```

```

### Reproducibility

It's a good idea to specify a pipeline version when running the pipeline on your data. This ensures that a specific version of the pipeline code and software are used when you run your pipeline. If you keep using the same tag, you'll be running the same version of the pipeline, even if there have been changes to the code since.

> First, go to the [steffenlem/sradownloader releases page](https://github.com/steffenlem/sradownloader/releases) and find the latest version number - numeric only (eg. `1.3.1`). Then specify this when running the pipeline with `-r` (one hyphen) - eg. `-r 1.3.1`.

Author: ```suggestion
First, go to the [qbic-pipelines/sradownloader releases page](https://github.com/qbic-pipelines/sradownloader/releases) and find the latest version number - numeric only (eg. `1.3.1`). Then specify this when running the pipeline with `-r` (one hyphen) - eg. `-r 1.3.1`.
```
